lib/msal-browser/docs/cdn-usage.md

```diff
@@ -73,10 +80,11 @@ It is highly recommended to use SRI Hashes with CDN builds of MSAL.js to help se
 ```
 
 ### SRI Hash Notes
+
```

Comment:
Given the SRI hashes below won't be kept updated on this branch, we may consider removing the contents of this entire doc except for the note above and a link to the v2 branch if someone is looking for the v2 information. Can be done later if you wish.

Reply:
That's a good point. Let's do that later once we finalize our release branching strategy.